### DISCUSSION TITLE
转错钱了

### DISCUSSION THREAD

**Discussion by yanxushi** [filecoin-project/community-china#155] [社区公告]
你好 我把fil币转到Token Binance-Peg Filecoin里了 能找回来吗？

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich
